docs/modules/reference/pages/daemons/introduction.adoc

```diff
@@ -99,14 +99,14 @@ This can be helpful for devices that cannot be directly polled to check service
 | The Telemetry daemon collects sensor data sent from nodes to {page-component-title}.
 
 | Ticketer
-| The ticketing integration lets Horizon create trouble tickets in external systems.
+| The ticketing integration lets {page-component-title} create trouble tickets in external systems.
 
 | Tl1d
 | Enables reception of TL1 autonomous messages.
 
 | Trapd
-| Trap daemon provides an SNMP server that accepts SNMP traps that can be converted to events.
+| The trap daemon provides an SNMP server that accepts SNMP traps that can be converted to events.
 
 | Vacuumd
-| Vacuum daemon takes care of cleaning processes in the PostgreSQL database.
+| The vacuum daemon takes care of cleaning processes in the PostgreSQL database.
```

Comment:
"takes care" ... does this mean "manages"?

Reply:
I would assume so. @mmahacek, would that be an accurate change?

Reply:
Vacuumd runs a set of tasks on a scheduled interval (every 24hrs from service start by default).  I wouldn't consider it managing as there isn't much intelligence in deciding what to run each time.
```suggestion
| The vacuum daemon executes daily cleanup tasks for the PostgreSQL database.
```